app/controllers/declaration/AdditionalInformationRequiredController.scala

```diff
@@ -69,9 +69,12 @@ class AdditionalInformationRequiredController @Inject()(
       )
   }
 
-  private def cachedItems(itemId: String)(implicit request: JourneyRequest[AnyContent]): Seq[AdditionalInformation] =
+  private def cachedAdditionalInformation(itemId: String)(implicit request: JourneyRequest[AnyContent]): Seq[AdditionalInformation] =
```

Comment:
This `def` could be maybe moved to `ExportsDeclaration` to be of general use. WDYT?